test/unit/accessibility/describe.js

```diff
@@ -0,0 +1,140 @@
+suite('describe', function() {
+  let myp5;
+  let myp5Container;
+  let myID = 'myCanvasID';
+  let a = 'a';
+  let b = 'b';
+  let c = 'c';
+
+  setup(function(done) {
+    myp5Container = document.createElement('div');
+    document.body.appendChild(myp5Container);
+    new p5(function(p) {
+      p.setup = function() {
+        let cnv = p.createCanvas(100, 100);
+        cnv.id(myID);
+        myp5 = p;
+        done();
+      };
+    }, myp5Container);
+  });
+
+  teardown(function() {
+    myp5.remove();
+    if (myp5Container && myp5Container.parentNode) {
+      myp5Container.parentNode.removeChild(myp5Container);
+    }
+    p5Container = null;
+  });
+
+  const getInner = elID => {
+    return document.getElementById(elID).innerHTML;
+  };
+
+  suite('p5.prototype.describe', function() {
+    let expected = 'A.';
+    test('should be a function', function() {
+      assert.ok(myp5.describe);
+      assert.typeOf(myp5.describe, 'function');
+    });
+    test('should create description as fallback', function() {
+      myp5.describe(a);
+      let actual = getInner(myID + '_fds');
+      assert.deepEqual(actual, expected);
+    });
+    test('should create description when called after describeElement()', function() {
+      myp5.describeElement(b, c);
+      myp5.describe(a);
+      let actual = getInner(myID + '_fds');
+      assert.deepEqual(actual, expected);
+    });
+    test('should create Label adjacent to canvas', function() {
+      myp5.describe(a, myp5.LABEL);
+      let actual = getInner(myID + '_lds');
+      assert.deepEqual(actual, expected);
+    });
+    test('should create Label adjacent to canvas when label of element already exists', function() {
+      myp5.describeElement(b, c, myp5.LABEL);
+      myp5.describe(a, myp5.LABEL);
+      let actual = getInner(myID + '_lds');
+      assert.deepEqual(actual, expected);
+    });
+    test('wrong param type at #0', function() {
+      assert.validationError(function() {
+        myp5.describe(1, myp5.LABEL);
+      });
+    });
+    test('no params', function() {
+      assert.validationError(function() {
+        myp5.describe();
+      });
+    });
+    test('err when LABEL at param #0', function() {
+      assert.throws(
+        function() {
+          myp5.describe(myp5.LABEL);
+        },
+        Error,
+        'description should not be LABEL or FALLBACK'
+      );
+    });
+  });
+
+  suite('p5.prototype.describeElement', function() {
+    let expected = '<th scope="row">A:</th><td>B.</td>';
```

Comment:
Great! as above: add a test case where the : and . are already present, and any other edge cases that matter.